ingest/scripts/link-gisaid-metadata-and-fasta

```diff
@@ -216,3 +216,6 @@ if __name__ == '__main__':
     sequences = pyfastx.Fasta(args.sequences)
     linked_records = link_metadata_and_sequences(metadata, sequences)
     dump_ndjson(linked_records)
+
+    # Remove Pyfastx index to save disk space
```

Comment:
How big is the pyfastx index? I've never used this myself...

Reply:
Definitely depends on the size of the original FASTA file. I've seen anywhere from 1MB to 7.5MB in the flu ingest.